src/main/java/org/broadinstitute/hellbender/tools/exome/SegmentUtils.java

```diff
@@ -280,6 +281,32 @@ public static SimpleInterval trimInterval(final Locatable interval,
         }
     }
 
+    /** Read only the sample names from a segment reader.
+     *
+     * @param segmentReader Never {@code null}
+     * @param segmentName Never {@code null}
+     * @return a list of the unique sample names in the seg file.
+     */
+    public static List<String> readSampleNamesFromSegmentReader(final Reader segmentReader, final String segmentName) {
```

Comment:
is it possible to rewrite the `readSampleNamesFromSegmentFile` method above to just call this method? Also what is the the `segmentName` input?